util/io/files/tar.go

```diff
@@ -177,7 +177,12 @@ func (t *tgz) tgzFile(path string, fi os.FileInfo, err error) error {
 		if err != nil {
 			return fmt.Errorf("error opening file %q: %w", fi.Name(), err)
 		}
-		defer f.Close()
+		defer func() {
+			err := f.Close()
+			if err != nil {
+				panic(fmt.Sprintf("error closing file %q: %v", fi.Name(), err))
```

Comment:
I think in this case we can just log the error rather than panicking.

Reply:
Thanks, modified, is this ok?

Reply:
Yes, thank you!